backends/platform/sdl/macosx/appmenu_osx.mm

```diff
@@ -135,7 +135,7 @@ - (void)openNews {
 }
 
 - (void)openUserManual {
-	[[NSWorkspace sharedWorkspace] openURL:[NSURL URLWithString:@"https://www.scummvm.org/documentation"]];
+	[[NSWorkspace sharedWorkspace] openURL:[NSURL URLWithString:@"https://docs.scummvm.org/"]];
```

Comment:
We could probably improve that to link directly the the version of the manual that corresponds to the ScummVM version. But this link is a bit outside the scope of this PR. I am mostly making a note for myself here.